File: QuevedoVirtualClassrooms-Server/src/main/java/com/quevedo/virtualclassroomsserver/common/models/dto/user/UserGetDTO.java
package com.quevedo.virtualclassroomsserver.common.models.dto.user;

import com.quevedo.virtualclassroomsserver.common.models.common.UserType;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class UserGetDTO {
    private String username;
    private String name;
    private String surname;
    private String email;
    private String profileImage;
    private UserType userType;
}

File: QuevedoVirtualClassrooms-Android/app/src/main/java/org/quevedo/quevedovirtualclassrooms/data/models/user/UserGetDTO.kt
package org.quevedo.quevedovirtualclassrooms.data.models.user

import org.quevedo.quevedovirtualclassrooms.data.models.common.UserType

data class UserGetDTO (
    private val username: String,
    private val name: String,
    private val surname: String,
    private val email: String,
    private val profileImage: String,
    private val userType: UserType
        )